File: CodeForWin/Patterns/Exercise 08/Solution.c
#include <stdio.h>


int main(){
  //User input
  int rows;
  printf("Input rows number: ");
  scanf("%d", &rows);

  //Print rows
  for (int i = 1; i <= rows; i++){
    //Print stars
    for (int x = 1; x <= i; x++){
      printf("*");
    }
    // New line
    printf("\n");
  }

  return 0;
}

File: CodeForWin/Patterns/Exercise 08/Solution.py
#!/usr/bin/env python3

# Function to print the triangle
def print_triangle(rows):
    length_rows = rows + 1
    for row in range(1, rows + 1):
        for star in range(1, row + 1):
            print("*", end = '')
        print("")

# Main function
def main():
    try:
        rows = int(input("Input row number: "))
        print_triangle(rows)

    except ValueError:
        print("[!] Invalid input")

#Run main function if the script is local
if __name__ == '__main__':
    main()
